CHANGELOG.adoc

```diff
@@ -130,7 +130,7 @@ image:doc/screenshots/showDeletionConfirmation.png[Deletion Confirmation Dialog,
 - https://github.com/eclipse-sirius/sirius-web/issues/3193[#3193] [core] Added the methods `hasVariable(String name)` and `getParent()` in VariableManager to, respectively, detect if a variable manager defines a variable (it returns `false`, if the variable is defined by a parent `VariableManager`) and to navigate to a parent `VariableManager`.
 Thanks to this capability, a specifier can easily retrieve a variable overriden by a child `VariableManager`.
 - https://github.com/eclipse-sirius/sirius-web/issues/3096[#3096] [tree] Tree representations can now support filters
-
+- https://github.com/eclipse-sirius/sirius-web/issues/3201[#3201] [diagram] Add support for tools on multiple diagram elements (hide, fade and pin).
```

Comment:
Add support for tools on multiple diagram elements selection (hide, fade and pin).

Reply:
done